com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitGUI.cs

```diff
@@ -265,7 +265,11 @@ public static void SetupMainTexForAlphaTestGI(this Material material, string col
             if (material.HasProperty(colorMapPropertyName))
             {
                 var mainTex = material.GetTexture(colorMapPropertyName);
+                var mainTexScale = material.GetTextureScale(colorMapPropertyName);
+                var mainTexOffset = material.GetTextureOffset(colorMapPropertyName);
                 material.SetTexture("_MainTex", mainTex);
+                material.SetTextureScale("_MainTex", mainTexScale);
+                material.SetTextureOffset("_MainTex", mainTexOffset);
```

Comment:
I also did that because URP does it, but couldn't find confirmation it was necessary. It just seems more logical